### PR TITLE
Remove JavaScript Compilation from the Frontend

### PR DESCRIPTION
**Pull request by Ferotiq**
### Description

Closes #

### CODE REVIEW

**Comment by Bruception** [comment]
Fully TS???

**Comment by Ferotiq** [comment]
for frontend, yes

File: frontend/webpack/config.prod.js [hunk 1]
```diff
@@ -52,20 +52,20 @@ const PRODUCTION_CONFIG = {
           flags: "g",
         },
       },
-      {
```
Comment (Bruception) [inline frontend/webpack/config.prod.js:55]: We need this.

Comment (Ferotiq) [reply frontend/webpack/config.prod.js:55]: ight

**Review by Miodec — APPROVED** [review]

**Pull request merged**